### DISCUSSION TITLE
【部署帮助】博客白屏，以及如何部署多个网站

### DISCUSSION THREAD

**Discussion by ptw1n1** [tangly1024/NotionNext#2164] [Q&A]
**描述bug**
部署完成后可以访问，但修改之后就成了光头页面了，有头有题，无内容。

**复现步骤**
在notion里面修改了友链、头像和一篇文章 而且多刷新了几下

**期望的正常结果**
正常显示博客

**截图**
<img width="1292" alt="截屏2024-02-17 14 15 54" src="https://github.com/tangly1024/NotionNext/assets/139853363/95ff4e9a-5434-45d3-8768-8550af9856ac">

**环境**

- NotionNext版本 4.1.1
- 主题 Next
- 部署方案 vercel
- 操作系统: macOS
- 浏览器 Chrome

**Comment by tangly1024** [comment]
请提供修改的内容

**Comment by ptw1n1** [comment]
> 请提供修改的内容

友链、头像和一篇文章
https://dent-seeker-113.notion.site/72862ff87b584bc1bdb01d2103f07594?v=31aa4afa9077445396d5841c3be11d33&pvs=4

**Comment by tangly1024** [comment]
疑惑的是 为为什么主题是commerce  这个主题未曾公开 并且不在新版本中维护

**Comment by ptw1n1** [comment]
> 疑惑的是 为为什么主题是commerce 这个主题未曾公开 并且不在新版本中维护

这个主题没有公开过吗？是个彩蛋吗 我也不大清楚，我就是在差不多十月跟着您的博客来操作的，然后随便改了一个主题。所以请问怎么修复呢

**Comment by tangly1024** [comment]
Commerce 是一个彩蛋主题； 

https://github.com/tangly1024/NotionNext/tree/feat/theme-commerce/themes/commerce；

很遗憾，由于并未合并到主分支，最新版本中暂时无法使用


除非暂时不升级到最新版本

**Comment by ptw1n1** [comment]
> Commerce 是一个彩蛋主题；
> 
> [https://github.com/tangly1024/NotionNext/tree/feat/theme-commerce/themes/commerce；](https://github.com/tangly1024/NotionNext/tree/feat/theme-commerce/themes/commerce%EF%BC%9B)
> 
> 很遗憾，由于并未合并到主分支，最新版本中暂时无法使用
> 
> 除非暂时不升级到最新版本

那请问修改主题就可以了嘛

**Comment by tangly1024** [comment]
是的，请尝试修改主题

**Comment by ptw1n1** [comment]
> 是的，请尝试修改主题

能教一下吗 忘记了怎么修改了 谢谢大佬！

**Comment by tangly1024** [comment]
> > 请提供修改的内容
> 
> 友链、头像和一篇文章 https://dent-seeker-113.notion.site/72862ff87b584bc1bdb01d2103f07594?v=31aa4afa9077445396d5841c3be11d33&pvs=4



在您给出的这个notion数据库中修改配置中心即可： 

![image](https://github.com/tangly1024/NotionNext/assets/15920488/f781a123-4718-4b21-b225-2c7952cf1a6f)

修改后直接刷新页面生效。

目前支持以下主题：

![image](https://github.com/tangly1024/NotionNext/assets/15920488/01034df2-67df-4bbb-b111-a28838b6fb29)

**Comment by ptw1n1** [comment]
非常感谢大佬～

**Comment by ptw1n1** [comment]
感谢大佬，成功了。我还有一个问题，请问可以创建多个副本（博客）吗

**Comment by tangly1024** [comment]
> 感谢大佬，成功了。我还有一个问题，请问可以创建多个副本（博客）吗

可以创建无数个

**Comment by ptw1n1** [comment]
> > 感谢大佬，成功了。我还有一个问题，请问可以创建多个副本（博客）吗
> 
> 可以创建无数个

请问如何创建？

**Comment by tangly1024** [comment]
可参考：https://docs.tangly1024.com/article/vercel-multi-sites

**Comment by ptw1n1** [comment]
那请问我使用GitHub登陆vercel的，那如果解绑GitHub会怎么样？那该怎么办

**Reply by tangly1024** [reply]
这里似乎是两个问题 ； 
1. Github账户登录Vercel账号能否解绑Github？能解绑， 在vercel 后台 account设置中可以解绑，但是没必要
2. Vercel 中从Github导入的项目能否断开与Github代码的绑定？可以，一份github代码只能被三个Vercel项目绑定，解绑有助于释放额度。
